samples/core/condition/condition_test.py

```diff
@@ -12,11 +12,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
+
+import unittest
 import kfp
+import kfp_server_api
+from ml_metadata.proto import Execution
 from .condition import my_pipeline
-from ...test.util import run_pipeline_func, TestCase
+from .condition_v2 import my_pipeline as my_pipeline_v2
```

Comment:
Can we rename the function name `my_pipeline` to `condition_pipeline`?

Reply:
Good point!